charts/firefly-signer/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v2
+name: firefly-signer
+description: |
+  A Helm chart for deploying the FireFly Signer microservice to Kubernetes
+type: application
+version: 0.7.0
+appVersion: "1.1.13"
```

Comment:
which are these version not matching FF version

Reply:
This is the correct version that was bundled with 1.3.0 https://github.com/hyperledger/firefly-signer/pkgs/container/firefly-signer/versions?filters%5Bversion_type%5D=tagged